### PR TITLE
Integrate the BeagleBone with Jenkins + pytest

### PR DESCRIPTION
**Pull request by GregorioDiStefano**
Included is all the code required to setup testing on the BBB with Jenkins/pytest. 
I've included some helpful scripts that aren't required for testing, but are facilitate setting up the environment (fabfile.py) and recovery (prepare_sdimg_testing.sh)

@kacf @pasinskim please take a look

### CODE REVIEW

File: tests/acceptance/files/fabfile.py [hunk 1]
```diff
@@ -0,0 +1,18 @@
+#!/usr/bin/python
+
+#Fabric file to copy over needed files to the BBB's internal system
+from fabric.api import put, sudo, settings, env
+
+
+def setup():
```
Comment (kacf) [inline tests/acceptance/files/fabfile.py]: Perhaps something a little more descriptive than just `setup`, like `setup_test_host`?

File: tests/acceptance/files/rssh.service [hunk 2]
```diff
@@ -0,0 +1,16 @@
+[Unit]
+Description=Reverse SSH tunnel
+Wants=network-online.target
+After=network-online.target
+
+[Service]
+Type=simple
+User=root
+Group=root
+ExecStart=/usr/bin/ssh -NTC -o ServerAliveInterval=60 -o ExitOnForwardFailure=yes -o StrictHostKeyChecking=no -R 12345:localhost:22 bbb@wmd.no
```
Comment (kacf) [inline tests/acceptance/files/rssh.service]: Document key stuff, like we talked about.

Comment (GregorioDiStefano) [inline tests/acceptance/files/rssh.service]: My bad, this should have included a README in order to clarify some things

**Comment by GregorioDiStefano** [comment]
@kacf @pasinskim 
Updated

**Comment by GregorioDiStefano** [comment]
@kacf @pasinskim can I go ahead and merge this?

**Comment by kacf** [comment]
Yep, looks good to me.

**Pull request merged**